main.tex

```diff
@@ -215,368 +222,312 @@ \subsection*{Functional variation}
 %
-We then computed the genetic distance ($D_{XY}$) between each of our sampled haplotypes and each of the inferred ancestral outbreak haplotypes, computing the distance in 2000bp windows (step size 1000bp) across a 5 Mbp region spanning the \textit{Vgsc} gene.
+Previous studies have found evidence for different rates of EHH decay between \texttt{L995F} and \texttt{L995S} haplotypes, which could be due to differences in haplotype ages and/or selection coefficients.
 %
-The results for clusters F1-F5 are plotted in Figure \ref{fig:recom_f}, S1-S4/5 are shown in Figure \ref{fig:recom_s} and L1-L2 in Figure \ref{fig:recom_l}.
-%%
-
-
-%% Figure - L995F recombination
+However, we found no significant difference in EHH decay between these two alleles (@@TODO data).
 %
-\begin{figure}[!b]
-  \includegraphics[width=1.1\linewidth,center]{artwork/fig_recom_F.png}
-  \caption{\textbf{Recombination and ancestral haplotypes for L995F}. @@TODO legend}
-  \label{fig:recom_f}
-\end{figure}
-%%
-
-
-%% Figure - L995S recombination
+The L1 haplotype carrying \texttt{I1527T+V402L} exhibited a slow decay of EHH on the downstream flank of the gene, similar to haplotypes carrying \texttt{L995F} and \texttt{L995S}, indicating that this combination of alleles has experienced positive selection.
 %
-\begin{figure}[!b]
-  \includegraphics[width=1.1\linewidth,center]{artwork/fig_recom_S.png}
-  \caption{\textbf{Recombination and ancestral haplotypes for L995S}. @@TODO legend}
-  \label{fig:recom_s}
-\end{figure}
-%%
-
-
-%% Figure - L995L recombination
+EHH decay on the upstream gene flank was faster, being similar to wild-type haplotypes, however we know from the analyses described above that there is evidence for recombination between \texttt{I1527T} and \texttt{V402L} within this group of haplotypes, and thus a faster decay of EHH is expected upstream of the core locus. 
 %
-\begin{figure}[!b]
-  \includegraphics[width=1.1\linewidth,center]{artwork/fig_recom_L.png}
-  \caption{\textbf{Recombination and ancestral haplotypes for L995L}. @@TODO legend}
-  \label{fig:recom_l}
-\end{figure}
+The L2 haplotype carrying \texttt{M490I} exhibited EHH decay intermediate between wild-type haplotypes and haplotypes carrying known resistance alleles.
+%
+This could indicate evidence for selection on the \texttt{M490I} allele, however these haplotypes are derived from a Kenyan mosquito population which is known to have experienced a severe recent bottleneck @@REF, and there are no wild-type haplotypes from Kenya with which to compare, thus this signal may also be due to the extreme demographic history of this population.
 %%
 
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Design of genetic assays for surveillance of pyrethroid resistance}
+
```

Comment:
I've edited the subsection on genetic assay design, no major changes, just added some preamble to provide a bit more context and motivation.